src/cryo_challenge/power_spectrum_utils.py

```diff
@@ -0,0 +1,71 @@
+import torch
+
+
+def _cart2sph(x, y, z):
```

Comment:
I would write some docstring (perhaps for the whole file at the top) about the conventions assumed here (origin, etc)